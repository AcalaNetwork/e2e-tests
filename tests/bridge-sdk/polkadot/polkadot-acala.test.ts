import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'polkadot',
  to: 'acala',
  token: 'DOT',
})
