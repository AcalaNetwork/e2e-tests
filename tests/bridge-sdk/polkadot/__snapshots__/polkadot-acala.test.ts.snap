// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'polkadot' to 'acala' using bridgeSDK cross-chain 'DOT' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 10,
  "fromChain": "(rounded 998)",
  "toChain": "(rounded 1000)",
}
`;

exports[`'polkadot' to 'acala' using bridgeSDK cross-chain 'DOT' > fee 1`] = `"(rounded 0.00008)"`;

exports[`'polkadot' to 'acala' using bridgeSDK cross-chain 'DOT' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 10,
  "fromChain": 1000,
  "toChain": 1000,
}
`;
